### DISCUSSION TITLE
Default hashing driver should be argon

### DISCUSSION THREAD

**Discussion by ajcastro** [laravel/framework#37507] [Ideas]
I got my code reviewed by a consultant. They told me to use argon instead of bcrypt. They gave me related article which has link below on bcrypt vs argon. I suggest argon should be the default. 
Is there any tradeoffs or issues? 
https://medium.com/analytics-vidhya/password-hashing-pbkdf2-scrypt-bcrypt-and-argon2-e25aaf41598e

**Comment by dinhquochan** [comment]
Argon2i no ship by default, then i prefer to bcrypt!

**Comment by eusonlito** [comment]
Argon2 is available as Laravel Hashing Method.

Check docs: https://laravel.com/docs/8.x/hashing#configuration
